docs/troubleshooting.md

```diff
@@ -103,6 +103,22 @@ If you run into issues where running `npx react-native init` hangs in your syste
 npx react-native init --verbose
 ```
 
+When you're debugging a process or need to know a little more about the error being thrown, they may want to use the verbose option to dump out more logs and information to nail down their issue.
+
+Run the following command to your root directory.
+
+For android
+
+```shell
+npx react-native run-android --verbose
+```
+
+For iOS
+
+```shell
+npx react-native run-ios --verbose
+```
```

Comment:
Can we merge them into a single snippet?

```
// Android
npx react-native run-android --verbose
// iOS
npx react-native run-ios --verbose
```

Reply:
We can do something like this,

```shell
npx react-native run-android --verbose && npx react-native run-ios --verbose
```

Also, we can add note like this :

If you only need to run for android, remove `&& npx react-native run-ios --verbose` and for iOS, remove `npx react-native run-android --verbose &&`

Reply:
> If you only need to run for android, remove `&& npx react-native run-ios --verbose` and for iOS, remove `npx react-native run-android --verbose &&`

It's safe to assume that the user knows that `run-android` should be replaced to `run-ios` if they're running an iOS app so I won't be so detailed

Reply:
Yes, it's easy to assume,  so we can do only this,

```shell
npx react-native run-android --verbose
```

Reply:
Yup that would also work